src/main/java/github/CollaboratorServiceEx.java

```diff
@@ -0,0 +1,47 @@
+package github;
+
+import org.apache.logging.log4j.Logger;
+import org.eclipse.egit.github.core.IRepositoryIdProvider;
+import org.eclipse.egit.github.core.User;
+import org.eclipse.egit.github.core.client.GitHubClient;
+import org.eclipse.egit.github.core.service.CollaboratorService;
+import org.eclipse.egit.github.core.service.UserService;
+import util.HTLog;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CollaboratorServiceEx extends CollaboratorService {
+
+    private static final Logger logger = HTLog.get(PullRequestServiceEx.class);
+
+    private final UserService userService = new UserService();
+
+    public CollaboratorServiceEx(GitHubClient client) {
+        super(client);
+    }
+
+    /**
+     * Gets the list of collaborators using CollaboratorService
+     * Then iterate through the list to get each collaborator using USerService
+     * This is done because the GitHub response for GET /repos/:owner/:repo/collaborators
+     * does not return a User with all the attributes of the user
```

Comment:
Maybe this part could be a bit clearer. It returns a user without all normal attributes/with attributes missing?